root_orchestrator/cloud-scheduler/resource_management/resource_helper.py

```diff
@@ -0,0 +1,8 @@
+from _datetime import datetime
+
+CLUSTERS_FRESHNESS_INTERVAL = 30
```

Comment:
Ideally if the resource abstractor is the one responsible for the resources, it should be his job to tell us if a cluster is active or not. In my opinion, we cannot ask each scheduler to have its own cluster-freshness logic.